### PR TITLE
Adding info in Configuring a Persistent Cart doc

### PR DESCRIPTION
**Pull request by dnahirna**
## Purpose of this pull request

This pull request describes adding missing info in the doc, fields:
Persist Wish List
Persist Recently Ordered Items
Persist Currently Compared Products
Persist Comparison History
Persist Recently Viewed Products
Persist Customer Group Membership and Segmentation

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [x] Other feature set, please specify: Customers -> Persistent Shopping Cart

## Affected documentation pages

https://docs.magento.com/user-guide/sales/cart-persistent-configuration.html

## Links to Magento source code or PRs

https://docs.magento.com/user-guide/configuration/customers/persistent-shopping-cart.html

### CODE REVIEW

**Comment by erikmarr** [comment]
Hi @dnahirna thank you for your contribution! Instead of adding a new version of an existing image with a new filename, please replace the existing image with your new image and keep the old filename.

**Comment by dnahirna** [comment]
Hello!

Thanks for your review
That was my initial thought but I just afraid to delete initial files

Best regards,
Daria Nahirna



чт, 19 нояб. 2020 г. в 21:19, Erik Marr <notifications@github.com>:

**Comment by dnahirna** [comment]
> Hi @dnahirna thank you for your contribution! Instead of adding a new version of an existing image with a new filename, please replace the existing image with your new image and keep the old filename.

Done

File: src/sales/cart-persistent-configuration.md [hunk 1]
```diff
@@ -47,4 +47,25 @@ If the session cookie expires while the customer is logged in, the persistent co
    Yes|If the session cookie expires, the persistent cookie is preserved. If a guest shopper later logs in or creates a new account, the shopping cart is restored.
    No|The shopping cart is not preserved for guests after the session cookie expires.
 
+1. **<span class="ee-only">Persist Wish List</span>** determines if the state of customer wish lists is retained when the session ends. Options:
```
Comment (leslietilling) [inline src/sales/cart-persistent-configuration.md:50]: For list items, span is not needed and simple liquid tag is best

```suggestion
1. {:.ee-only}Set **Persist Wish List** to determine if the state of customer wish lists is retained when the session ends:
```

File: src/sales/cart-persistent-configuration.md [hunk 2]
```diff
@@ -47,4 +47,25 @@ If the session cookie expires while the customer is logged in, the persistent co
...
+
+   Yes|The wish list contents are saved when the session ends.|
+   No|The wish list is not saved when the session ends.|
+
+1. **<span class="ee-only">Persist Recently Ordered Items</span>** determines if the state of recently ordered items is retained when the session ends. Options:
```
Comment (leslietilling) [inline src/sales/cart-persistent-configuration.md:55]:
```suggestion
1. {:.ee-only}Set **Persist Recently Ordered Items** to determine if the state of recently ordered items is retained when the session ends:
```

File: src/sales/cart-persistent-configuration.md [hunk 3]
```diff
@@ -47,4 +47,25 @@ If the session cookie expires while the customer is logged in, the persistent co
...
+   Yes|The state of Recently Ordered Items is saved when the session ends.|
+   No|The state of Recently Ordered Items is not saved when the session ends.|
+
+1. Set **Persist Currently Compared Products** to one of the following: `Yes` or `No`
+
+1. Set **Persist Comparison History** to one of the following: `Yes` or `No`
+
+1. Set **Persist Recently Viewed Products** to one of the following: `Yes` or `No`
+
+1. <span class="ee-only">Persist Customer Group Membership and Segmentation</span> determines if the state of customers’ group membership and segmentation criteria are retained when the session ends. Options:
```
Comment (leslietilling) [inline src/sales/cart-persistent-configuration.md:66]:
```suggestion
1. {:.ee-only}Set **Persist Customer Group Membership and Segmentation** to determine if the state of customers’ group membership and segmentation criteria are retained when the session ends:
```

File: src/sales/cart-persistent-configuration.md (hunk 1 above)
Comment (dnahirna) [reply src/sales/cart-persistent-configuration.md:50]: done

File: src/sales/cart-persistent-configuration.md (hunk 2 above)
Comment (dnahirna) [reply src/sales/cart-persistent-configuration.md:55]: done

File: src/sales/cart-persistent-configuration.md (hunk 3 above)
Comment (dnahirna) [reply src/sales/cart-persistent-configuration.md:66]: done

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**